infra.lvn_tools/extend_boot/roles/extend_boot/files/module-setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+check(){
+    return 0
+}
+
+install() {
+    inst_multiple -o /usr/sbin/parted /usr/bin/mktemp /usr/bin/wc /usr/bin/date /usr/bin/sed /usr/bin/awk /usr/bin/basename /usr/sbin/resize2fs /usr/sbin/tune2fs /usr/sbin/partprobe /usr/bin/numfmt /usr/sbin/lvm /usr/bin/lsblk /usr/sbin/e2fsck /usr/sbin/fdisk /usr/bin/findmnt /usr/bin/tail
+    inst_hook pre-mount 99 "$moddir/extend-boot-partition.sh"
+    inst_binary "$moddir/sfdisk.static" "/usr/sbin/sfdisk"
+    inst_simple "$moddir/extend.sh" "/usr/bin/extend.sh"
+}
```

Comment:
Please add an empty line

Reply:
I'm curious, why add the empty line in a bash script at the end? I don't see shellcheck complaining about it so I wonder if there is another reason I'm missing.